core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/MisoRequestManager.java

```diff
@@ -1185,6 +1186,31 @@ public void saveRuns(Collection<Run> runs) throws IOException {
     }
   }
 
+  private void setValuesBeforeSavingNewRun(Run run) throws IOException {
```

Comment:
This method does the same as `saveRun` when the run is unsaved. Can just call `saveRun` for all runs in `saveRuns` instead of separating them